setup.py

```diff
@@ -80,35 +77,16 @@ def is_tool(name):
     # TODO: Modifying this should allow to remove the MAINFEST.in
     include_package_data=True,
     install_requires=requirements + [
-        "pywin32==225; sys_platform=='win32'"
+        "pywin32==225; sys_platform=='win32'",
+        "pythonnet==2.5.2; sys_platform=='win32'"
```

Comment:
Shouldn't this be the other way around? pythonnet is required if NOT windows? The pywin requirement is also an annoying one btw... We should define this in a `requirements_system.txt` or something (also in alphatemplate)